apps/site/src/app/login/components/LoginForm.tsx

```diff
@@ -1,31 +1,61 @@
-import ValidatingForm from "@/lib/components/ValidatingForm/ValidatingForm";
+"use client";
 
-const EMAIL_REGEX = /^\w+([.-]?\w+)*@\w+([.-]?\w+)*(\.\w{2,3})+$/;
+import { FormEvent, useState } from "react";
+
+import RequiredAsterisk from "@/app/apply/sections/Components/RequiredAsterisk";
+import Button from "@/lib/components/Button/Button";
+
+const EMAIL_REGEX = /^\w+([\.\-]?\w+)*@\w+([\.\-]?\w+)*(\.\w{2,3})+$/;
 const LOGIN_PATH = "/api/user/login";
 
 function LoginForm() {
+	const [validated, setValidated] = useState<boolean | null>(null);
+
+	const handleSubmit = (event: FormEvent<HTMLFormElement>): void => {
+		const form = event.currentTarget;
+		if (!form.checkValidity()) {
+			// prevent submission to display validation feedback
+			event.preventDefault();
+			setValidated(false);
+		} else {
+			setValidated(true);
+		}
+	};
+
 	return (
-		<ValidatingForm method="post" action={LOGIN_PATH}>
-			<div>
-				<label>Email address</label>
+		<form
+			noValidate
+			method="post"
+			action={LOGIN_PATH}
+			onSubmit={handleSubmit}
+			className="bg-white p-10 rounded-2xl mx-5 text-black"
+		>
+			<div className="flex flex-col mb-5">
+				<label htmlFor="email">
+					Email <RequiredAsterisk />
```

Comment:
Thought/suggestion: required asterisk feels redundant when there is only one input field.